config/settings.yml

```diff
@@ -14,3 +14,100 @@ issue_manager:
   repo_names: []
 travis_event:
   enabled_repos: []
+trello:
+  # https://github.com/jeremytregunna/ruby-trello#configuration
+  oauth:
+    consumer_key:
+    consumer_secret:
+    oauth_token:
+    oauth_token_secret:
+  basic_auth:
+    developer_key:
+    member_token:
+  organization_id:
+  teams:
```

Comment:
Ah, right ...

So, I have them locally in `development.local.yml`.  I can change this to be more generic (e.g., "team1, team2", etc...).